loader/loader.go

```diff
@@ -29,6 +29,8 @@ import (
 	"github.com/open-policy-agent/opa/util"
 )
 
+const readUNCPathEnv = "READ_UNC_PATH"
```

Comment:
I'm not a huge fan of shadow config via env vars 😅 But if we have to do it, let's prefix this with `OPA_`